src/components/dashboard/ActionsList.js

```diff
@@ -216,7 +222,32 @@ const ActionIcon = ({ actionKind }) => (
 )
 
 const ActionTimeStamp = ({ timeStamp }) => (
-    <div className='font-small time-stamp'>{format(timeStamp)}</div>
+    <div className='font-small'>{format(timeStamp)}</div>
+)
+
+const TX_STATUS = {
+    NotStarted: {
+        textId: 'notStarted',
```

Comment:
why introduce separate `textId` instead of just using tx status name as is?

Reply:
I thought about it but figured that if status name will be changed, it will be easier to change it in one place, than in a few places in translation files.

Reply:
>  if status name will be changed

blockchain API has to be frozen with mainnet release, any changes would be complex multi-step process

> than in a few places in translation files.

seems like simple search&replace in multiple files, no need for extra mapping